kube_hunter/conf/__init__.py

```diff
@@ -14,6 +14,7 @@
 parser.add_argument('--report', type=str, default='plain', help="set report type, options are: plain, yaml, json")
 parser.add_argument('--dispatch', type=str, default='stdout', help="where to send the report to, options are: stdout, http (set KUBEHUNTER_HTTP_DISPATCH_URL and KUBEHUNTER_HTTP_DISPATCH_METHOD environment variables to configure)")
 parser.add_argument('--statistics', action="store_true", help="set hunting statistics")
+parser.add_argument('--discovery-timeout', type=int, default=5, help="kube-apiserver discovery timeout")
```

Comment:
Describe this argument as general timeout for discovery operations rather than specifically kube-apiserver